src/core/apim.tf

```diff
@@ -340,15 +340,15 @@ resource "azurerm_api_management_api_version_set" "apim_external_api_ms" {
 
 module "apim_external_api_ms_v1" {
   source              = "git::https://github.com/pagopa/azurerm.git//api_management_api?ref=v2.12.5"
-  name                = format("%s-ms-external-api-v1", local.project)
+  name                = format("%s-ms-external-api", local.project)
```

Comment:
the `-v1` suffix was removed because it will be added automatically (otherwise it will end with `-v1-v1` suffix)